insights/parsers/readlink_openshift_certs.py

```diff
@@ -0,0 +1,60 @@
+"""
+ReadLinkEKubeletClientCurrent - command and others``
```

Comment:
Since there is another parser `ReadLinkEKubeletServerCurrent` in this module, better modify the title correspondingly.

Reply:
Is this better?
```Commands: ReadLinkEKubeletClientCurrent and ReadLinkEKubeletServerCurrent```